### PR TITLE
Remove xarray pin in environment; call dask compute on the two data a…

### PR DESCRIPTION
**Pull request by ktyle**
…rrays; current version of Panel fixes time slider in rasterized dataset

### CODE REVIEW

**Comment by brian-rose** [comment]
For some reason the preview never deployed. Rerunning the build to see if that fixes it.

**Comment by brian-rose** [comment]
I ran the notebooks and build the cookbook locally no problem. It's not clear to me what's failing on GitHub. Will try closing and reopening aka unplugging the router.

**Comment by brian-rose** [comment]
There must be a package conflict in the sphinx / jupyter-book stack. I will investigate elsewhere.

**Comment by brian-rose** [comment]
So this is frustrating. I used the exact same environment description in [this PR on the Cookbook template](https://github.com/ProjectPythia/cookbook-template/pull/130) and did not reproduce the same failure.

Somehow the two builds are ending up different despite having the same contents in `environment.yml`. My guess is that the collision between mamba and pip is to blame. We use pip to install `sphinx-pythia-theme`, which is a workaround (https://github.com/ProjectPythia/cookbook-template/issues/6) waiting for resolution of https://github.com/ProjectPythia/sphinx-pythia-theme/issues/39.

At the end of the [`Update execution environment` step in this PR](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:334), the log says
```
Installing collected packages: sphinx-book-theme
[334](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:335)
  Attempting uninstall: sphinx-book-theme
[335](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:336)
    Found existing installation: sphinx-book-theme 1.0.1
[336](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:337)
    Uninstalling sphinx-book-theme-1.0.1:
[337](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:338)
      Successfully uninstalled sphinx-book-theme-1.0.1
[338](https://github.com/ProjectPythia/ERA5_interactive-cookbook/actions/runs/5414254894/jobs/9841049947?pr=7#step:15:339)
Successfully installed sphinx-book-theme-0.1.10
```

Over at https://github.com/ProjectPythia/cookbook-template/pull/130 the [same stage of the build log](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:534) says
```
Installing collected packages: docutils, pydata-sphinx-theme, sphinx-book-theme, sphinx-pythia-theme
[534](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:535)
  Attempting uninstall: docutils
[535](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:536)
    Found existing installation: docutils 0.17.1
[536](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:537)
    Uninstalling docutils-0.17.1:
[537](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:538)
      Successfully uninstalled docutils-0.17.1
[538](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:539)
  Attempting uninstall: pydata-sphinx-theme
[539](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:540)
    Found existing installation: pydata-sphinx-theme 0.13.3
[540](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:541)
    Uninstalling pydata-sphinx-theme-0.13.3:
[541](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:542)
      Successfully uninstalled pydata-sphinx-theme-0.13.3
[542](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:543)
  Attempting uninstall: sphinx-book-theme
[543](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:544)
    Found existing installation: sphinx-book-theme 1.0.1
[544](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:545)
    Uninstalling sphinx-book-theme-1.0.1:
[545](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:546)
      Successfully uninstalled sphinx-book-theme-1.0.1
[546](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:547)
Successfully installed docutils-0.16 pydata-sphinx-theme-0.7.2 sphinx-book-theme-0.1.10 sphinx-pythia-theme-2022.3.29
[547](https://github.com/ProjectPythia/cookbook-template/actions/runs/5414625261/jobs/9841953808#step:15:548)
```

Basically it looks like pip is downgrading several packages in the sphinx stack to satisfy the requirement to use an old version of `sphinx-book-theme`. And that is working for most cookbook.

Why is the same thing not happening in this PR? I have no idea right now.

**Comment by brian-rose** [comment]
I'll try pinning those packages in the environment file here so mamba installs the correct versions. But it would be great to understand what's causing the failure and/or how to reproduce it.

**Comment by brian-rose** [comment]
Pinning to 
```
  - pydata-sphinx-theme=0.7.2
  - sphinx-book-theme=0.1.10
```
caused a different build failure when I ran it locally.

Not sure where to go from here.

**Comment by ktyle** [comment]
Very interesting/confounding, since I believe the trigger-book-build action succeeded last night after I modified 01BasicVisualization.ipynb with the necessary dask.compute call. 

Definitely makes the need to modernize the sphinx-pythia-theme more immediate.
